ctx.go

```diff
@@ -146,14 +161,16 @@ type netHttpContext[Body any] struct {
 }
 
 var (
-	_ ContextWithBody[any]    = &netHttpContext[any]{}    // Check that ContextWithBody implements Ctx.
-	_ ContextWithBody[string] = &netHttpContext[string]{} // Check that ContextWithBody implements Ctx.
-	_ ValidableCtx            = &netHttpContext[any]{}    // Check that ContextWithBody implements ValidableCtx.
+	_ Context[string, struct{}] = &netHttpContext[string, struct{}]{} // Check that ContextWithBody implements Ctx.
+	_ Context[any, any]         = &netHttpContext[any, any]{}         // Check that ContextWithBody implements Ctx.
+	_ ContextWithBody[any]      = &netHttpContext[any, any]{}         // Check that ContextWithBody implements Ctx.
+	_ ContextWithBody[string]   = &netHttpContext[string, any]{}      // Check that ContextWithBody implements Ctx.
+	_ ValidableCtx              = &netHttpContext[any, any]{}         // Check that ContextWithBody implements ValidableCtx.
```

Comment:
I think there is a lot of copy pasta right here maybe with the comments? Probably should go above as well.

Reply:
Fixed!